Hamburger.js

```diff
@@ -0,0 +1,59 @@
+const MenuItem = require('./MenuItem').MenuItem;
+// const Stuffing = require('./Stuffing').Stuffing;
```

Comment:
Please remove the comments from all the documents. 
![comments_infographic-01edit_3](https://user-images.githubusercontent.com/53054365/122607777-c4718800-d083-11eb-9278-59933f3f168c.jpeg)